boards/remote-pa/board.c

```diff
@@ -60,7 +60,7 @@ static void rf_switch_init(void)
     RF_SWITCH_PORT->DIR |= (1 << RF_SWITCH_PIN);
 
     /* configure io-mux for used pins */
-    IOC->OVER[RF_SWITCH_PIN] = IOC_OVERRIDE_OE;
+    IOC_PXX_OVER[RF_SWITCH_PIN] = IOC_OVERRIDE_OE;
```

Comment:
there is no direct benefit other than making the code consistent, because currently both variants are used, i.e., `IOC_PXX_OVER` and `IOC_PXX_SEL` map to directly to address while `IOC` uses a struct with members `OVER` and `SEL`. 

Anyway, I simply want to get rid of one or the other to make the code more consistent.